src/sentry/spans/consumers/process/flusher.py

```diff
@@ -139,8 +155,41 @@ def poll(self) -> None:
         self.next_step.poll()
 
     def submit(self, message: Message[FilteredPayload | int]) -> None:
+        # Note that submit is not actually a hot path. Their message payloads
+        # are mapped from *batches* of spans, and there are a handful of spans
+        # per second at most. If anything, self.poll() might even be called
+        # more often than submit()
+        if not self.process.is_alive():
+            metrics.incr("sentry.spans.buffer.flusher_dead")
+            if self.process_restarts < MAX_PROCESS_RESTARTS:
+                self.process.start()
+                self.process_restarts += 1
+            else:
+                raise RuntimeError("flusher process is dead")
```

Comment:
```suggestion
                raise RuntimeError("flusher process has crashed")
```

Ideally we would have the inner error linked here, which is not trivial. We can note in the runbook to check for errors from the flusher process. Can we initialize the SDK with a special tag within the flusher process so this gets easier?